sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.aest.reissueMSFAA.e2e-spec.ts

```diff
@@ -0,0 +1,355 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../testHelpers";
+import {
+  E2EDataSources,
+  MSFAAStates,
+  createE2EDataSources,
+  createFakeMSFAANumber,
+  saveFakeApplication,
+  saveFakeApplicationDisbursements,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import {
+  ApplicationStatus,
+  DisbursementScheduleStatus,
+  OfferingIntensity,
+} from "@sims/sims-db";
+
+describe("ApplicationAESTController(e2e)-reissueMSFAA", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should reissue an MSFAA and associate with both disbursements when both disbursements are pending and the current MSFAA is signed but canceled.", async () => {
+    // Arrange
+    const student = await saveFakeStudent(db.dataSource);
+    const currentMSFAA = createFakeMSFAANumber(
+      { student },
+      {
+        state: MSFAAStates.Signed | MSFAAStates.CancelledOtherProvince,
```

Comment:
It is as good as using `state:MSFAAStates.Signed` right?  or am I missing something?